jboss-eap-standalone-rhel8/README.md

```diff
@@ -24,12 +24,11 @@
 
 JBoss EAP is an open source platform for highly transactional, web-scale Java applications. EAP combines the familiar and popular Jakarta EE specifications with the latest technologies, like Microprofile, to modernize your applications from traditional Java EE into the new world of DevOps, cloud, containers, and microservices. EAP includes everything needed to build, run, deploy, and manage enterprise Java applications in a variety of environments, including on-premise, virtual environments, and in private, public, and hybrid clouds.
 
-This template deploys a web applicaton deployed on JBoss EAP 7.2 running on RHEL 8. 
+This template creates all of the compute resource to run JBoss EAP 7 on top of RHEL 8.0.
```

Comment:
I have added the name of the application here, please let me know whether I should remove it.

This Azure quickstart template creates all of the compute resources to run a web application named JBoss-EAP on Azure on JBoss EAP 7.2 running on top of RHEL 8.0 VM.